### PR TITLE
added message definitions for reading limit switches over can

### PR DESCRIPTION
**Pull request by caila-marashaj**

### CODE REVIEW

File: include/motor-control/core/motion_controller.hpp [hunk 1]
```diff
@@ -57,6 +57,10 @@ class MotionController {
 
     void stop() { hardware.stop_timer_interrupt(); }
 
+    auto read_limit_switch() -> uint8_t {
+        return static_cast<uint8_t>(hardware.check_limit_switch());
```
[the remark this answers is not in the document]
Comment (caila-marashaj) [reply include/motor-control/core/motion_controller.hpp:61]: it returns a boolean in the motor hardware interface

[the remark this answers is not in the document]
Comment (amitlissack) [reply include/motor-control/core/motion_controller.hpp:61]: If casting to `uint8_t` is for satisfying the CAN message, then I suggest that `MotorController::read_limit_switch` return a `bool`  and let the cast to `uint8_t` happen in `handle` below.

[the remark this answers is not in the document]
Comment (sfoster1) [reply include/motor-control/core/motion_controller.hpp:61]: +1

**Review by sfoster1 — CHANGES_REQUESTED** [review]
Oh we should add the python side here but this does look good

**Comment by caila-marashaj** [comment]
> Oh we should add the python side here but this does look good

I believe we took care of this in this pr:
https://github.com/Opentrons/ot3-firmware/pull/221

**Review by amitlissack — APPROVED** [review]
Thank you. Nice!

**Review by sfoster1 — APPROVED** [review]
In that case, great! Looks good to me when checks pass

**Pull request merged**